servers/plugins/org.fusesource.ide.server.fuse.core/src/org/fusesource/ide/server/fuse/core/bean/ServerBeanTypeFuse6x.java

```diff
@@ -34,7 +34,7 @@
 	
 	protected ServerBeanTypeFuse6x() {
 		super(	"FUSE6x", //$NON-NLS-1$
-				"JBoss Fuse 6.x", //$NON-NLS-1$
+				"Red Hat Fuse 6.x", //$NON-NLS-1$
```

Comment:
6.x need to be kept as-is https://issues.jboss.org/browse/FUSETOOLS-2843?focusedCommentId=13545002&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-13545002